requirements/test-mindeps.isolated

```diff
@@ -0,0 +1,23 @@
+# FIXME
+# this version of requests is newer than the minimum in package data
+# however, if we pin back to 2.19.1, we break 'responses' behaviors used by the
+# testsuite
+requests==2.22.0
+
+pyjwt==2.0.0
+cryptography==3.3.1
+typing_extensions==4.0
+
+## replicate the test dependencies here, rather than including the test.in file
+## if included, pip-compile-multi fails
+##
+##   Package idna was resolved to different versions in different environments: 2.7 and 3.4
+##   Traceback (most recent call last):
+##     ...
+##   RuntimeError: Please add constraints for the package version listed above
+##
+#pytest
+#coverage
+#pytest-xdist
+#responses
+-r test.in
```

Comment:
Could this be at the top of the file? Even if pip-compile-multi / pip-tools interprets this correctly, I'd prefer for this file to look like its explicit versions are overriding the contents of `test.in`.

Reply:
Oops... This is a development artifact. I was trying it both ways with pip-compile and pip-compile-multi.
pip-compile-multi was able to handle things if we listed the packages twice, which was interesting behavior to explore but not something I was very comfortable using.

I'll remove the whole commented out section.